aries_cloudagent/core/conductor.py

```diff
@@ -690,19 +710,19 @@ def _queue_internal(
             return OutboundSendStatus.QUEUED_FOR_DELIVERY
         except OutboundDeliveryError:
             LOGGER.warning("Cannot queue message for delivery, no supported transport")
-            return self.handle_not_delivered(profile, outbound)
+            return await self.handle_not_delivered(profile, outbound)
 
-    def handle_not_delivered(
+    async def handle_not_delivered(
         self, profile: Profile, outbound: OutboundMessage
     ) -> OutboundSendStatus:
         """Handle a message that failed delivery via outbound transports."""
         queued_for_inbound = self.inbound_transport_manager.return_undelivered(outbound)
-
-        return (
+        status = (
```

Comment:
This change doesn't seem necessary anymore since we don't do anything else with status before returning it